src/components/BrokerView/index.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from "react";
+import { Link } from "react-router-dom";
+
+import {
+  IServiceBinding,
+  IServiceBroker,
+  IServiceClass,
+  IServiceInstance,
+  IServicePlan,
+} from "../../shared/ServiceCatalog";
+import { Card, CardContainer } from "../Card";
+import SyncButton from "../SyncButton";
+
+export interface IBrokerViewProps {
+  bindings: IServiceBinding[];
+  broker: IServiceBroker | undefined;
+  classes: IServiceClass[];
+  getCatalog: () => Promise<any>;
+  instances: IServiceInstance[];
+  plans: IServicePlan[];
+  sync: (broker: IServiceBroker) => Promise<any>;
+}
+
+export class BrokerView extends React.PureComponent<IBrokerViewProps> {
+  public async componentDidMount() {
+    this.props.getCatalog();
+  }
+
+  public render() {
+    const { bindings, broker, instances } = this.props;
+
+    return (
```

Comment:
This render method is too long. Table elements and card containers could be extracted to specific components.